src/plugins/date/README.md

```diff
@@ -98,4 +98,5 @@ TBD
 
 ## Limitations ##
 
-None.
+Testing timezone designators currently only works with glibc.
+Unit tests using timezone designators are locale dependent.
```

Comment:
Please describe in `Limitations` the limitations the user has (not that we have with the test cases).